gooddata/load_faa_postgres.py

```diff
@@ -1,32 +1,33 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-import os
 import re
 from pathlib import Path
 from fastparquet import ParquetFile
 from pandas import DataFrame
 import logging
 from databases.postgres import PostgresDbConfig, Postgres
-
+from libs.data_source_config import DataSourceConfig
+from libs.config import default_workspace_id
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
-project = "faa"
 entities = [
     "aircraft_models", "aircraft", "airports", "carriers", "flights"
 ]
 data_dir = Path("..") / "malloy" / "data"
-ddl_dir = Path(".") / "ddl" / project
-transform_dir = Path(".") / "transform" / project
-db_host = os.getenv('DB_HOST', 'localhost')
+ddl_dir = Path(".") / "ddl" / default_workspace_id
+transform_dir = Path(".") / "transform" / default_workspace_id
```

Comment:
Could you please use something similar to what we have in the Python SDK repo instead of `Path(".")`?

`_current_dir = Path(__file__).parent.absolute()`